golang/internal/health/types.go

```diff
@@ -0,0 +1,10 @@
+package health
+
+const (
+	socketType = "unix"
+	socketPath = "/tmp/agenthealth.sock"
```

Comment:
A better path would be: path.Join(os.TempDir(),"dyrectorio","agenthealth.sock"), pay attention to file existence privilege error handling too.